Components: Add ValidateControls components (#70620)
WordPress/gutenberg: packages/components/src/validated-form-controls/components/stories/overview.mdx

import { ArgTypes, Meta, Stories } from '@storybook/blocks';
import * as OverviewStories from './overview.story';
import { ValidatedInputControl } from '..';

<Meta of={ OverviewStories } title="Components (Private)/Validated Form Controls/Overview" />

# Validated Form Controls

This section contains form control components that extend [WordPress components](https://wordpress.github.io/gutenberg/) with additional validation capabilities.

## Status: Beta

> We are still gathering feedback and iterating. Please get in touch with Team Calypso if you are interested in trying them out in your project.

Reviewer: ```suggestion
We are still gathering feedback and iterating. Please get in touch with `@WordPress/gutenberg-components`  if you are interested in trying them out in the Gutenberg app.
```

Author: I missed this! Thanks! Fixed with b80006d9a1c9d4b4ae371f6ebad1f2b234276edf
